cpp/src/umap/runner.h

```diff
@@ -426,6 +426,8 @@ void _transform(const cumlHandle &handle, T *X, int n, int d,
     params->callback->on_preprocess_end(transformed);
   }
 
+  params->initial_alpha /= 4.0;
```

Comment:
Just curious. Is this part of the original algorithm?

Reply:
Oops. Actually I think this was added more recently but I didn’t mean to check that  change in here. I happened to be looking through the code while working on the umap paper this weekend and wanted to see if it had any effect on tthe results